src/NuGet.Core/NuGet.PackageManagement/NuGetPackageManager.cs

```diff
@@ -3041,7 +3075,8 @@ var projectUniqueNamesForBuildIntToUpdate
                     restoreResult,
                     sources.ToList(),
                     nuGetProjectActionsList,
-                    installationContext);
+                    installationContext,
+                    nuGetProjectActions.First().VersionRange);
```

Comment:
Is nuGetProjectActions guaranteed to have at least one item every time this code path hits? If so, could you point me at where this is enforced or deterministic?

Reply:
In this same function we use `nuGetProjectActions.First().PackageIdentity` thats why I though it was right to use it, but looking at my code I already have the versionRange as a parameter for this function, I will use that instead